services/web/client/source/class/qxapp/About.js

```diff
@@ -102,7 +80,7 @@ qx.Class.define("qxapp.About", {
       entryLayout.add(entryLabel);
 
       const text14Font = qx.bom.Font.fromConfig(qxapp.theme.Font.fonts["text-14"]);
```

Comment:
I would suggest creating a custom appearance or even widget to have consistency in the UI instead of local formatting.